test/util.ts

```diff
@@ -31,6 +31,13 @@ export function parseUnitModelWithScale(spec: TopLevel<UnitSpec>) {
   return model;
 }
 
+export function parseUnitModelWithScaleAndLayoutSize(spec: TopLevel<UnitSpec>) {
```

Comment:
Another method would be making all parse support chaining -- however I am not sure if we can do that yet since we are not done with all merging stuff.  

In any case, these helpers method make it way easier to find and replace test methods so I won't waste my precious time refactoring them more.